spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/publisher/PubSubPublisherOperations.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright 2017-2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.pubsub.core.publisher;
+
+import java.util.Map;
+
+import com.google.pubsub.v1.PubsubMessage;
+
+import org.springframework.util.concurrent.ListenableFuture;
+
+/**
+ * An abstraction for Google Cloud Pub/Sub publisher operations.
+ *
+ * @author Vinicius Carvalho
+ * @author João André Martins
+ * @author Mike Eltsufin
+ * @author Chengyuan Zhao
+ * @author Doug Hoard
+ * @since 1.1.0
+ */
+public interface PubSubPublisherOperations {
+
+	/**
+	 * Send a message to Pub/Sub.
+	 *
```

Comment:
No blank lines in the method Javadocs.

Reply:
Removed blank lines from Javadoc comments (both in files I changed as well as the original source files in `org.springframework.cloud.gcp.pubsub.core`)

Reply:
@dhoard ,

Thank you very much for the quick turn around incorporating our code style nit-picks, but I still have one concern:
How about do not comment that you have done the fix here and there?
If you agree with us, you just fix comment in the commit message and push!

This way I will only receive a single email message 😄